### ISSUE TITLE
Show nested content item buttons on focus

### ISSUE THREAD

**Issue by bjarnef** [umbraco/Umbraco-CMS#7285]
I noticed when using keyboard to navigation nested content, it doesn't show the copy and delete button when these are focused. It is because they are only shown on hover.

Maybe these should be shown when the nested content header bar is focused or at least when focus is set on each of these buttons.

![image](https://user-images.githubusercontent.com/2919859/70316506-2957eb00-181c-11ea-8562-da02bd5c5fdc.png)

---
_This item has been added to our backlog [AB#4535](https://dev.azure.com/umbraco/243e7927-03b2-44e2-908f-d4ac7ea5daaa/_workitems/edit/4535)_

**Comment by bjarnef** [comment]
@nielslyngsoe what do you think of this? I think it would be great to show these buttons (copy and delete) when `umb-nested-content__header-bar` has focus.

**Comment by nielslyngsoe** [comment]
@bjarnef I was supposed to implement the ":focus-within" pseudo-class, but now investigating I see it's not been implemented, so we should do that.

And we should also have a look atthat label jumping...

Thanks for mentioning this, if you have a fix for the label please feel free to make a PR for it and then I will review it straight away :-)

**Comment by nielslyngsoe** [comment]
And I have made a PR: https://github.com/umbraco/Umbraco-CMS/pull/7287

**Comment by nielslyngsoe** [comment]
@bjarnef could you make a new Issue for the jumping label, and if you have a good suggestion on how to solve already, then please provide it as PR :-D
Thanks

**Comment by bjarnef** [comment]
@nielslyngsoe just had a look at in seems this fix this issue, but it might be a bigger change to switch the part to flexbox.

```
.umb-control-group .umb-el-wrap {
    display: flex;
    flex-flow: row wrap;
}

.form-horizontal .umb-control-group .control-header {
    flex: 0 0 160px;
}

.form-horizontal .controls {
    flex: 1 1 auto;
}
```

I think the issue here is that the "placeholder" shown when dragging is higher than the "real" items.

**Comment by bjarnef** [comment]
@nielslyngsoe I have created a new issue for the "jumping" label.
https://github.com/umbraco/Umbraco-CMS/issues/7288

**Comment by kjac** [comment]
Fixed by #7287
